python-api/src/api/llm/router.py

```
@@ -0,0 +1,447 @@
+    await websocket.accept()
+    logger.info("WebSocket /llm/ws/chat connected")
+
+    history: list[dict[str, str]] = []
+
+    try:
+        while True:
+            raw = await websocket.receive_text()
+
+            try:
+                frame = WSIncomingMessage.model_validate_json(raw)
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)